docs/intro/using.rst

```diff
@@ -86,47 +85,127 @@ is set using the utility function :meth:`~hybrid.utils.min_sample`.
+['SPIN', 2 rows, 2 samples, 6 variables]
 
 Such :class:`.Branch` classes can be run in parallel using the :class:`.RacingBranches` class.
 From the outputs of these parallel branches, :class:`.ArgMin` selects a new current sample.
 And instead of a single iteration on the sample set, you can use the :class:`.Loop`
 to iterate a set number of times or until a convergence criteria is met.
 
-This next example solves a binary quadratic model by iteratively producing best samples.
+This example of :ref:`racingBranches1` solves a binary quadratic model by iteratively producing best samples.
 Similar to :std:doc:`qbsolv <qbsolv:index>`, it employs both tabu search on the entire
 problem and a D-Wave system on subproblems. In addition to building-block components
 such as employed above, this example also uses infrastructure classes to manage the
 decomposition and parallel running of branches.
 
+.. figure:: ../_static/racing_branches_1.png
+  :name: racingBranches1
+  :scale: 70 %
+  :alt: Racing Branches
+
+  Racing Branches
 
 .. include:: ../../README.rst
   :start-after: example-start-marker
   :end-before: example-end-marker
 
+Flow Refining
+-------------
+
+The framework enables quick modification of work flows to improve solutions and performance.
+For example, after verifying the :ref:`racingBranches1` workflow above on its small problem,
+you might make a series of modifications such as the examples below to better fit it to problems
+with large numbers of variables.
+
+1. Configure a decomposition window that moves down a fraction of problem variables,
+   ordered from highest to lower energy impact, and submit those subproblems to the D-Wave system
+   while tabu searches globally. This example submits 50-variable subproblems on
+   up to 15% of the total variables.
+
+.. code-block:: python
+
+    # Redefine the workflow: a rolling decomposition window
+    subproblem = hybrid.EnergyImpactDecomposer(size=50, rolling_history=0.15)
+    subsampler = hybrid.QPUSubproblemAutoEmbeddingSampler() | hybrid.SplatComposer()
+
+    iteration = hybrid.RacingBranches(
+        hybrid.InterruptableTabuSampler(),
+        subproblem | subsampler
+    ) | hybrid.ArgMin()
+    workflow = hybrid.LoopUntilNoImprovement(iteration, convergence=3)
+
+2. Instead of sequentially producing a sample per subproblem, a further modification might be to
+   process all the subproblems in parallel and merge the returned samples. Here the
+   :class:`~hybrid.decomposers.EnergyImpactDecomposer` is iterated until it raises a
+   :meth:`~hybrid.exceptions.EndOfStream` exception when it reaches 15% of the variables,
+   and then all the 50-variable subproblems are submitted to the D-Wave system.
+
+.. code-block:: python
+
+    # Redefine the workflow: parallel subproblem solving for a single sample
+    subproblem = hybrid.Unwind(
+                 hybrid.EnergyImpactDecomposer(size=50, rolling_history=0.15, silent_rewind=False))
```

Comment:
No need for `silent_rewind=False` inside `Unwind` anymore (since #120).

Reply:
Removed